src/fontra/views/applicationsettings/panel-shortcuts.js

```diff
@@ -0,0 +1,682 @@
+      }
+    });
+    return shortCutDefinition;
+  }
+
+  recordShortCut(id, event) {
+    event.preventDefault();
+
+    const pressedKey = this.getPressedKey(event);
+    this.pressedKeys.add(pressedKey);
+
+    const shortCutDefinition = this.getShortCutDefinition();
+
+    // show the current shortcut immediately, no delay:
+    const element = document.getElementById(id);
+    element.value = getShortCutRepresentation(shortCutDefinition);
+
+    //if not alt, shift, ctrl or meta, end of recording -> save shortcut
+    if (!event[pressedKey]) {
+      if (!this.saveShortCuts([shortCutDefinition])) {
+        // if the shortcut is invalid, reset the input field
+        element.value = getShortCutRepresentation(this.shortCutDefinition);
+      }
+      element.blur(); // remove focus
+      this.pressedKeys = new Set();
+    }
+  }
+
+  recordShortCutKeyup(id, event) {
+    // This removes the unpressed key
+    this.pressedKeys.delete(this.getPressedKey(event));
+
+    const element = document.getElementById(id);
+    const shortCutRepresentation = getShortCutRepresentation(
+      this.getShortCutDefinition()
+    );
+    element.value =
+      shortCutRepresentation != ""
+        ? shortCutRepresentation
+        : getShortCutRepresentation(this.shortCutDefinition);
+  }
+
+  resetShortCut(id) {
+    setCustomShortCuts(this.key, null);
+
+    document.getElementById(id).value = getShortCutRepresentation(
+      getShortCuts(this.key)[0]
+    );
+    this.shortCutDefinition = getShortCut(this.key);
+  }
+
+  deleteShortCut(id) {
+    if (this.saveShortCuts([null])) {
+      document.getElementById(id).value = "";
+    }
+  }
+
+  _updateContents() {
+    this.innerHTML = "";
+    this.append(
+      html.label(
+        {
+          class: "fontra-ui-shortcuts-panel-label",
+        },
+        [this.shortCutLabel]
+      )
+    );
+
+    const id = `shortcut-input-${this.key}`;
+    this.append(
+      html.input({
+        type: "text",
+        id: id,
+        class: "fontra-ui-shortcuts-panel-input",
+        value: getShortCutRepresentation(this.shortCutDefinition),
+        // tooltip does not work with text input element -> use title instead.
+        title:
+          "Click and record a shortcut OR double click and open dialog for editing",
+        onkeydown: (event) => this.recordShortCut(id, event),
+        onkeyup: (event) => this.recordShortCutKeyup(id, event),
+        ondblclick: (event) => this.doEditShortCut(id),
+      })
+    );
+
+    this.append(
+      html.createDomElement("icon-button", {
+        "class": "fontra-ui-shortcuts-panel-icon",
+        "src": "/tabler-icons/refresh.svg",
+        "onclick": (event) => this.resetShortCut(id),
+        "data-tooltip": "Reset to default",
+        "data-tooltipposition": "top",
+      })
+    );
+
+    this.append(
+      html.createDomElement("icon-button", {
+        "class": "fontra-ui-shortcuts-panel-icon",
+        "src": "/tabler-icons/trash.svg",
```

Comment:
I think we should use an "X" icon instead: https://tabler.io/icons/icon/x

Reply:
done.